apps/publications/src/app/pages/my-publications-page/my-publications-page.component.ts

```diff
@@ -12,26 +12,28 @@ import { TABLE_PUBLICATION_AUTHOR_DETAIL_PUBLICATIONS } from '@perun-web-apps/co
 import { ActivatedRoute } from '@angular/router';
 import { MatDialog } from '@angular/material/dialog';
 
+interface Filter {
```

Comment:
dont you want to move this interface to its single file, so we don't have to duplicate?

Reply:
removed duplicates